tmt/base.py

```diff
@@ -143,14 +143,17 @@ def fmf_id(self):
 
         def run(command):
             """ Run command, return output """
+            cwd = fmf_root if self.steps() == {'discover'} else None
```

Comment:
Q: Is there a situation where setting cwd to fmf_root will break stuff?
I don't like `self.steps() == {'discover'} ` - it is not easy to see why magic value is `{'discover'}` and imho this check is not necessary

Reply:
This code is responsible for the case mentioned by you below https://github.com/teemtee/tmt/pull/932#discussion_r821672134

Reply:
I know, but is there use case where cwd can't be set to fmf_root ? From what I tried cwd can be safely set to fmf_root each time and thus such check is not necessary